spoj_practice/spoj2/spoj2.py

```diff
@@ -51,12 +38,14 @@ def find_primes_between (n, m):
     # m = numbers[0]
     # n = numbers[1]
     # instead, do:
-    m, n = stdin.readline().split(' ')
-    m = int(m)
-    n = int(n)
+    m, n = map(int, stdin.readline().split(' '))
     prime = []
```

Comment:
Kind of. I don't think C++ quite works that way normally. Pointers aren't really a thing in Python, so it's hard to draw equivalents here. Anyway, yes, every time a new object is declared in Python, it's a completely new object. What you have to watch out for is whether or not the function you're calling creates a new object or modifies the existing one(s).